### DISCUSSION TITLE
influx DB I lost my dashboard

### DISCUSSION THREAD

**Discussion by isaacgolding** [Security-Onion-Solutions/securityonion#11089] [2.4]
So the influxDB screen that initially loads on 2.4... I was messing with things and mine went poof.   I'm quite sure it's me.   Short of a reinstall.... how do I bring that initial screen back?   Is there a JSON file somewhere I can load to bring back that initial view?

**Comment by dougburks** [comment]
You might take a look at `/usr/sbin/so-influxdb-manage`:
https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/salt/influxdb/tools/sbin/so-influxdb-manage
